zeppelin-zengine/src/main/java/org/apache/zeppelin/interpreter/remote/RemoteInterpreterManagedProcess.java

```diff
@@ -141,7 +141,7 @@ public void start(String userName) throws IOException {
 
   public void stop() {
     if (isRunning()) {
-      LOGGER.info("Kill interpreter process");
+      LOGGER.info("Kill interpreter process for interpreter group: " + getInterpreterGroupId());
```

Comment:
We should avoid the creation of objects if a logging level is disabled 
This particular line should change to
```
LOGGER.info("Kill interpreter process for interpreter group: {}", getInterpreterGroupId());
```